paper/paper.md

```diff
@@ -0,0 +1,113 @@
+    orcid: 0000-0003-0841-2707
+    affiliation: 1
+  - name: Helge Hecht
+    orcid: 0000-0001-6744-996X
+    affiliation: 1
+  - name: Martin Čech
+    orcid: 0000-0002-9318-1781
+    affiliation: 1
+  - name: Elliott James Price
+    orcid: 0000-0001-5691-7000
+    affiliation: 1
+
+affiliations:
+ - name: RECETOX, Faculty of Science, Masaryk University, Kotlarska 2, Brno 60200, Czech Republic
+   index: 1
+date: 11 January 2022
+bibliography: paper.bib
+ 
+---
+
+# Summary
+
+MSMetaEnhancer is a Python software package for annotation of mass spectra files. 
+The package uses matchms [@Huber2020] for data IO and supports `.msp` input and output data format.
+It annotates given mass spectra file by adding missing metadata such as SMILES, InChI, and CAS numbers.
+The package retrieves the medatada by querying several external databases, 
+currently supporting CIR, CTS [@Wohlgemuth2010], ChemIDplus [@tomasulo2002chemidplus], and PubChem [@kim2021pubchem].
+The package is hosted via bioconda and is available on Galaxy.
+
+# Statement of need
+
+Mass spectra data need to be annotated with metadata to make them more reproducible and interoperable with other datasets. 
+While this metadata is mostly accessible from public chemical databases, they are not always present in the output of the mass spectrometry libraries. 
+Therefore, the data needs to be post-processed and appropriate metadata gathered from reliable sources.
+Such a process usually cannot be fully automated, and assistance from the user is required to specify particular annotation steps and sources. 
+
+# State of the field
+
+There are several packages within Python and R ecosystems which support querying external
+databases. 
+For example, there are R packages that provide an interface to PubChem [@guha2016,@cao2008chemminer], and a package with interface to wikidata [@keys2021]. 
+Then, there are packages unifying several sources -- `webchem` [@szocs2020webchem] allows to automatically query chemical data from several web sources and `MetaFetcheR`[@yones2021metafetcher] links metabolite data from several small-compound databases, trying to resolve inconsistencies.
+
+On the Python side, there are packages for PubChem [@swain2017], ChemSpider [@swain2018], or CIR [@swain2016]. 
+However, to the best of out knowledge, there is no Python package connecting these sources into a single tool, allowing straightforward annotation of mass spectra files.
+
+# The software package
+
+MSMetaEnhancer is an annotation tool for mass spectra files.
+It takes as input a single `.msp` file with multiple mass spectra records and a list of annotation steps.
+These steps consist of specifying what service should be used to obtain a particular metadata attribute based on another already existing attribute.
+The supported service currently includes CIR, CTS, ChemIDplus, and PubChem.
+The supported metadata attributes are InChI, InChIKey, SMILES, IUPAC chemical name, chemical formula, and CAS number. 
```

Comment:
This section should be made more vague (like includes among others xzy) etc., as the package might evolve and support more services, formats etc. and we likely won't publish a new version of the paper any time soon - so it can be formulated in such a way so that it doesn't get outdated so fast.